src/Layout/redist/targets/GenerateMSIs.targets

```diff
@@ -40,6 +46,9 @@
     <LightCommandObjDir>$(ArtifactsObjDir)LightCommandPackages</LightCommandObjDir>
     <!-- Directory for the zipped up light command package -->
     <LightCommandPackagesDir>$(ArtifactsNonShippingPackagesDir)</LightCommandPackagesDir>
+
+    <!-- workaround so warnings-as-errors doesn't cause the missing manifest msi message to be treated as an error -->
+    <MSBuildWarningsNotAsErrors>$(MSBuildWarningsNotAsErrors);MISSING_MANIFEST_MSI</MSBuildWarningsNotAsErrors>
```

Comment:
Can you elaborate on why this is necessary? I'm not following.

Reply:
Shoudl this be conditioned on `AllowEmptyBuiltinWorkloadManifestMsi`?

Reply:
> Should this be conditioned on AllowEmptyBuiltinWorkloadManifestMsi?

Yeah we can do that.

> Can you elaborate on why this is necessary? I'm not following.

This is basically to be able to print a warning locally about the SDK bundle not containing the manifest msis without breaking the build. A normal `<Warning />` will run into warn-as-error and `<Message Importance="High" />` doesn't show up with the terminal logger... I agree it's a bit "clever" and we can remove it but then people building the bundle locally might not be aware of this.